test.txt

```diff
@@ -1,18 +1,16 @@
 You are now connected to database "nc_news_test" as user "jaysonchiu".
```

Comment:
You probably don't need to keep committing these files. 
Try to avoid defaulting to using shortcuts like `git add .` - instead only commit the files you intended to.